lib/webidl2.js

```diff
@@ -305,6 +305,14 @@
       return { value, trivia };
     }
 
+    function unvalue_token(token) {
```

Comment:
It will be used frequently but it's mainly to keep API compatibility. I would ultimately remove this and modify the getters just return booleans. Same for `untype_token()`.

Reply:
Makes sense.